wrangler-service/src/main/java/co/cask/wrangler/service/gcs/GCSHandler.java

```diff
@@ -325,9 +319,14 @@ public void loadObject(HttpServiceRequest request, HttpServiceResponder responde
         .setScope(scope)
         .setProperties(properties)
         .build();
-      ws.writeWorkspaceMeta(workspaceMeta);
+      if (blob.isDirectory()) {
+        throw new BadRequestException("Path specified is not a file.");
```

Comment:
should we add blobpath to error message? Having contextual info in error message would be better?

Reply:
fixed